src/components/Assignment/index.ts

```diff
@@ -275,7 +278,8 @@ class Assignment extends BridgeBase {
     this.navigateToStep = actionsAPI.navigateToStep.bind(actionsAPI);
     this.cancelAssignment = actionsAPI.cancelAssignment.bind(actionsAPI);
     this.showPage = actionsAPI.showPage.bind(actionsAPI);
-
+    this.localizedVal = PCore.getLocaleUtils().getLocaleValue;
+    this.localeReference = `${this.thePConn.getCaseInfo().getClassName()}!CASE!${this.thePConn.getCaseInfo().getName()}`.toUpperCase();
```

Comment:
Is this the only way to get the localization ref?  Would this not already be available (pre-constructed) from the view metadata?